### PR TITLE
QA Bug Fixes

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
Mantis#0860803 - "Add Item" action should have an example of an input to JSON
- Added relevant tooltip for the `Additional Attribute` field
- Updated the info.json with more input parameter

Mantis#0860802 - Playbook failing with ERROR:: Parameter validation failed
- Added `remove_unwanted_parameter` function to remove unnecessary parameters before calling AWS service API for DynamoDB

Mantis#0860448 - Connector Logo not visible clearly
- Replaced the logo file and validated that logo is visible and clearly

Mantis#0860443 - SSL check is missing from Connector configurations
- Added SSL check into the configuration

### CODE REVIEW

File: aws-dynamodb/operations.py [hunk 1]
```diff
@@ -10,115 +10,195 @@
...
 
+def remove_unwanted_param(params):
+    try:
+        params.pop('aws_region', None)
+        params.pop('assume_role', None)
+        params.pop('session_name', None)
+        params.pop('role_arn', None)
+        param_dict = {k: v for k, v in params.items() if v is not None and v != '' and v != {} and v != []}
+        return param_dict
+    except Exception as Err:
```
Comment (cs-prerna-joshi) [inline aws-dynamodb/operations.py]: remove the try- exception block to avoid the unnecessary the stack trace.

Comment (cs-rohit-khune) [reply aws-dynamodb/operations.py]: @cs-prerna-joshi  okay, from all the functions?

Comment (cs-prerna-joshi) [reply aws-dynamodb/operations.py]: yes.

Comment (cs-rohit-khune) [reply aws-dynamodb/operations.py]: @cs-prerna-joshi 
Done suggested changes. Please check

**Pull request merged**